sorting_test.go

```diff
@@ -71,33 +71,50 @@ func readLines(path string) ([]string, error) {
 	var lines []string
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
+		// ignore blank lines
+		d := scanner.Text()
+		if d == "" {
+			continue
+		}
 		lines = append(lines, scanner.Text())
 	}
 	return lines, scanner.Err()
 }
 
-func load() {
-	l = make([]int, 10000000)
-	lines, err := readLines("arr.txt") //your array file
+func load(filename string) ([]int, error) {
+	lines, err := readLines(filename)
```

Comment:
Same comment here, let's remove the filename and response parameters for semplicity .